src/main/scala/cognite/spark/v1/ThreeDModelRevisionsRelation.scala

```diff
@@ -36,3 +36,7 @@ class ThreeDModelRevisionsRelation(config: RelationConfig, modelId: Long)(val sq
       client: GenericClient[IO]): Seq[Stream[IO, ThreeDRevision]] =
     Seq(client.threeDRevisions(modelId).list(config.limitPerPartition))
 }
+
+object ThreeDModelRevisionsRelation extends NamedRelation {
+  override val name: String = "3dmodelrevisions"
```

Comment:
value is changing here, but looks like nothing depends on it in transformations atm :thinking:

Reply:
Maybe support wasn't added to transformations yet https://docs.cognite.com/cdf/integration/guides/transformation/write_sql_queries/ - not listed here

Reply:
does match the datasource `case` string, so better to stick to that, only shortname for metrics changes here like in the other 3d file :+1:

Reply:
Ah yes, I mentionned the other one but forgot this one changes too. It does look right though!